api/src/uvisor-lib.c

```diff
@@ -0,0 +1,42 @@
+/*
+ * Copyright (c) 2016, ARM Limited, All Rights Reserved
+ * SPDX-License-Identifier: Apache-2.0
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License"); you may
+ * not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+ * WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+#include "api/inc/export_table_exports.h"
+
+int uvisor_post_init(void)
+{
+    /* Supported */
+    /* Defined in uvisor-input.S */
+    extern uint32_t uvisor_config;
+    extern uint32_t uvisor_export_table_size;
+
+    uintptr_t uvisor_config_addr = (uintptr_t)&uvisor_config;
```

Comment:
Is there a value in having `uintptr_t` instead of `uint32_t` since anyway we then cast it to  `(TUvisorExportTable *)`?

You could have directly this:

```C
TUvisorExportTable * uvisor_export_table = (TUvisorExportTable *) ((uint32_t) &uvisor_config - uvisor_export_table_size);
```

(`uvisor_export_table_size` is already a `uint32_t`)